README.md

```diff
@@ -0,0 +1,34 @@
+# Scala-AkkaHTTP-REST-API
+
+Application to explore movies, screenings and to reserve seats
+
+To test type:
+`sbt test`
+
+To run type:
+`sbt run`
+
+While running use demo script:
+`sh demo.sh`
+
+Possible requests: 
+
+- to print movies screenings by given time period:
+
+`curl -X GET -H 'Content-Type: application/json' -i 'http://localhost:8080/movies?from=2020-12-01%2001:00:00&until=2020-12-09%2001:00:00'` 
```

Comment:
will it be sorted? In requirements we have `Screenings given in point 2. of the scenario should be sorted by title and screening
time.`

Reply:
owww, i forgot about that